lib/jekyll-seo-tag.rb

```diff
@@ -1,7 +1,12 @@
+require "jekyll"
 require "jekyll-seo-tag/version"
 
 module Jekyll
   class SeoTag < Liquid::Tag
+    autoload :JSONLD,  "jekyll-seo-tag/json_ld"
+    autoload :Drop,    "jekyll-seo-tag/drop"
+    autoload :Filters, "jekyll-seo-tag/filters"
```

Comment:
I recently ran into an issue with this because I didn't add `lib` to the `$LOAD_PATH`, so local tests didn't work.